### PR TITLE
Update travis build scripts 2019 12 20 for master

### PR DESCRIPTION
**Pull request by JonRowe**
@benoittgt @pirj this seems to be the minimum required changes to get our build green so I'm proposing we use this...

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -42,21 +42,12 @@ else
   gem 'sqlite3', '~> 1.3.6', platforms: [:ruby]
 end
 
-if RUBY_VERSION >= '2.4.0'
-  gem 'json', '>= 2.0.2'
-end
```
Comment (JonRowe) [inline Gemfile]: Technically these made no difference for this build.

File: spec/sanity_check_spec.rb [hunk 2]
```diff
@@ -34,7 +34,7 @@ def with_clean_env
     end
   end
 
-  it "passes when libraries are required" do
+  it "passes when libraries are required", unless: RSpec::Support::Ruby.jruby? do
```
Comment (benoittgt) [inline spec/sanity_check_spec.rb]: Do you think we should revert that change when gem/json-jruby warning will be fixed?

**Comment by benoittgt** [comment]
We have many open PR for the same thing (https://github.com/rspec/rspec-rails/pull/2239). I am happy to merge this one.

File: spec/sanity_check_spec.rb (hunk 2 above)
Comment (JonRowe) [reply spec/sanity_check_spec.rb]: Yes probably.

**Pull request merged**